test-it/karma/events/ControlEvents.Test.js

```diff
@@ -244,8 +246,17 @@ describe('The Vireo Control Event', function () {
 
         runSlicesAsync(function (rawPrint, rawPrintError) {
             expect(rawPrintError).toBeEmptyString();
-            expect(viPathParser('eventOccurred')).toEqual(true);
-            expect(viPathParser('eventTimedOut')).toEqual(false);
+            expect(rawPrint).toBe('Waiting on Events\nEvent occurred\n');
```

Comment:
Instead you can do:
```
var result = `Waiting on Events
Event occurred
`;
 expect(rawPrint).toMatchVtrText(result)
```

It cleans up line endings, removes comments, and prints a diff of the changes on failures. Used by the karma vtr runner.

Reply:
Actually, only really valuable for lots of text, so nvm